package.json

```diff
@@ -5,7 +5,7 @@
   "dependencies": {
     "babel-runtime": "^6.20.0",
     "bluebird": "~2.9.24",
-    "mobify-progressive-app-sdk": "~0.18.0"
+    "mobify-progressive-app-sdk": "1.0.0-beta4"
```

Comment:
Should this be `-rc1`?

https://github.com/mobify/astro/pull/801/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R3